contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/SearchOrBrowseWindow.vue

```diff
@@ -140,8 +142,12 @@
         clearNodes: 'CLEAR_NODES',
       }),
       handleBackToBrowse() {
+        this.languages = '';
```

Comment:
I had to retain it because the events only emit "languageFilter" changes. For instance, if I first searched for Spanish (es), and then returned to the Home Imports page, the language would still be set to the previous one. Consequently, when I click the "backToBrowse" link and search the query without any language filters (the filter remains present due to the previous language), the language settings need to revert to default. Were you able to understand the issue?